src/schema/fair.ts

```diff
@@ -144,6 +144,12 @@ export const FairType = new GraphQLObjectType<any, ResolverContext>({
       description: "A formatted description of the start to end dates",
       resolve: ({ start_at, end_at }) => exhibitionPeriod(start_at, end_at),
     },
+    dateStatusDisplay: {
```

Comment:
I think this field could have a more self-documenting name, something like ‘formatted opening status’?

Reply:
Yes I wasn't super happy with the name either, I'll fix.

Reply:
That being said it's not just opening but can be used to indicate it's closed so I may go with `formattedDateStatus`

Reply:
Perhaps it’s my non-native English (?), but I do think that eg “opening hours” could be “closed on Sunday”. If y’all think that the current iteration is a good self-documenting name, then I’m fine with it.

Reply:
Ok you've convinced me, I present thee `formattedOpeningHours`